src/server/helpers/entityRouteUtils.js

```diff
@@ -247,7 +247,7 @@ export function makeEntityCreateOrEditHandler(
 	entityType: string,
 	transformNewForm: Function,
 	propertiesToPick: string | string[],
-	isMergeHandler?: boolean = false
+	isMergeHandler: boolean = false
```

Comment:
Gives off an error in the linter when both `?` and a default value are provided.